src/mounts.rs

```diff
@@ -476,21 +481,31 @@ fn mask_path(path: &str) -> Result<()> {
         let msg = format!("invalid maskedPath: {}", path);
         return Err(ErrorKind::InvalidSpec(msg).into());
     }
-    if let Err(e) = mount(
+
+    let _ = mount(
         Some("/dev/null"),
         path,
         None::<&str>,
-        MS_BIND,
+        MsFlags::MS_BIND,
         None::<&str>,
-    )
-    {
-        // ignore ENOENT and ENOTDIR: path to mask doesn't exist
-        if e != Sys(Errno::ENOENT) && e != Sys(Errno::ENOTDIR) {
-            let msg = format!("could not mask {}", path);
-            Err(e).chain_err(|| msg)?;
+    ).map_err(|e| {
```

Comment:
There is something wrong with the logic here. It isn't ignoring enoent like it should